### ISSUE TITLE
Colors very bland after processing

### ISSUE THREAD

**Issue by rawtaz** [gruntjs/grunt-contrib-imagemin#162]
Hi,

So I'm trying out imagemin, and while it works fine for a lot of files, I've found that it can be pretty destructive on colors. Here are a couple of cuts from two of the image files (both JPG) I had processed:

![geo](https://f.cloud.github.com/assets/825204/2304835/f638c286-a237-11e3-9d1c-56370366440e.png) ![blue](https://f.cloud.github.com/assets/825204/2304836/fc291704-a237-11e3-900e-f856f0cca1a9.png)

In both examples the bottom part is the original, having the correct colors, and the upper part is the processed one, having bland colors.

So basically I'm wondering, what can we do about it? Is it even worth pursuing? It seems that the underlying libraries are pretty explicit in that they won't kill quality/lose information, so if this is happens anyway then perhaps it's not much to spend time on.

Mostly curious what others thing, perhaps I'm the only one who cares about my colors :)
